addon/components/base/bs-dropdown.js

```diff
@@ -214,7 +214,7 @@ export default Component.extend({
    */
   menuElement: computed(function() {
     return document.getElementById(`${this.get('elementId')}__menu`);
-  }),
+  }).volatile(),
```

Comment:
`.volatile()` has been deprecated, so we shouldn't use it. (in fact I removed a few of those instance recently because of this). As the tests are set up to fail for thrown deprecations, Travis is failing here.

Instead of this we should be able to use a standard ES5 getter, though it requires some awkward setup (until we move to native classes), see e.g. https://github.com/kaliber5/ember-bootstrap/blob/master/addon/components/base/bs-modal.js#L696-L725

Reply:
I've updated the code to use that syntax, thanks for the quick answer.